gm4_chairs/data/chairs/functions/kill.mcfunction

```diff
@@ -0,0 +1,7 @@
+# @s = @e[tag=gm4_chairs] and NOT inside stair blocks
+#execute at @s align xyz run tp @e[nbt={RootVehicle:{Entity:{id:"minecraft:pig"}}},dx=0,dz=0,dy=2] ~ ~.4 ~
+tp @s ~ ~.5 ~
+execute at @s run summon minecraft:area_effect_cloud ~ ~ ~ {Duration:100,Tags:["gm4_chairs_temp"]}
+kill @s

```

Comment:
by merging `Health:0b`, the entity won't drop porkchop, you have to summon the saddle back though but killing porkchop can kill the wrong one and stuff like that.